<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fabryka PCB</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayoutMain">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tabOrderData">
         <attribute name="title">
          <string>Zamówienia</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidgetOrders"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabBoardData">
         <attribute name="title">
          <string>Płytki</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidgetBoards"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabSheets">
         <attribute name="title">
          <string>Arkusze</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayoutSheets">
            <item>
             <widget class="QGraphicsView" name="graphicsView"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutSheetControl">
              <item>
               <widget class="QPushButton" name="buttonPrevSheet">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditCurrentSheet">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>/</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditMaxSheet">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonNextSheet">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_control">
         <attribute name="title">
          <string>Panel Sterowania</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_output">
         <attribute name="title">
          <string>Wyniki</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDane">
    <property name="title">
     <string>Dane</string>
    </property>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuOptymalizacja">
    <property name="title">
     <string>Optymalizacja</string>
    </property>
    <addaction name="actionAlgorytm_Prosty"/>
   </widget>
   <addaction name="menuDane"/>
   <addaction name="menuOptymalizacja"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionAlgorytm_Prosty">
   <property name="text">
    <string>Algorytm Prosty</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
